### DISCUSSION TITLE
WebOne 0.12.2

### DISCUSSION THREAD

**Discussion by atauenis** [atauenis/webone#67] [General]
New stable version of WebOne, released on 2 Sep 2022 after 6 commits.

* Added support for Keep-Alive connections to servers. This significantly improving speed of page loading. Some sites are now opening more than 2 times faster than via previous versions.
* Fixed processing of 404 errors and similar. Now if `SearchInArchive` option is enabled, WebOne will always try to load archived copies of "not found" pages. In v0.8.0-0.12.1 this feature worked not quite stable.
* Fixed installing in Ubuntu 22.04/22.10 on armhf/arm64 architectures.
* Dropped support for `InternalRedirectOn` list in configuration files. It was a rudiment from older versions, and has not been widely used.

<hr /><em>This discussion was created from the release <a href='https://github.com/atauenis/webone/releases/tag/v0.12.2'>WebOne 0.12.2</a>.</em>

**Comment by prosper00** [comment]
Thank you! I get great use out of webone, and and always excited each time I see a notification of a new release!

**Comment by way5** [comment]
Thank your for a new release @atauenis ! 
For those who need a docker image here is one https://github.com/way5/docker-webone

**Reply by atauenis** [reply]
Thank you. :)
